WordPress/src/main/java/org/wordpress/android/ui/prefs/categories/SiteSettingsCategoriesViewHolder.kt

```diff
@@ -0,0 +1,53 @@
+package org.wordpress.android.ui.prefs.categories
+
+import android.os.Bundle
+import androidx.core.view.ViewCompat
+import androidx.recyclerview.widget.RecyclerView
+import org.apache.commons.text.StringEscapeUtils
+import org.wordpress.android.R
+import org.wordpress.android.databinding.SiteSettingsCategoriesRowBinding
+import org.wordpress.android.models.CategoryNode
+import org.wordpress.android.ui.utils.UiDimen.UIDimenRes
+import org.wordpress.android.ui.utils.UiHelpers
+
+class SiteSettingsCategoriesViewHolder(
+    private val categoryBinding: SiteSettingsCategoriesRowBinding,
+    val uiHelpers: UiHelpers,
+    private val onClickListener: (CategoryNode) -> Unit
+) : RecyclerView.ViewHolder(categoryBinding.root) {
+    private val verticalPadding: Int = uiHelpers.getPxOfUiDimen(
+            categoryBinding.siteSettingsCategoryText.context,
+            UIDimenRes(R.dimen.margin_large)
+    )
+    private val horizontalPadding: Int = uiHelpers.getPxOfUiDimen(
+            categoryBinding.siteSettingsCategoryText.context,
+            UIDimenRes(R.dimen.margin_extra_large)
+    )
```

Comment:
Minor Suggestion: Drop explicit `Int` declaration and directly use `itemView.context`.

```suggestion
    private val verticalPadding = uiHelpers.getPxOfUiDimen(itemView.context, UIDimenRes(R.dimen.margin_large))
    private val horizontalPadding = uiHelpers.getPxOfUiDimen(itemView.context, UIDimenRes(R.dimen.margin_extra_large))
```

Reply:
Done in https://github.com/wordpress-mobile/WordPress-Android/pull/15580/commits/955f8bfb9c4d7344138f828edbce81231e3fb9da